docs/DeveloperGuide.md

```diff
@@ -137,60 +144,68 @@ This section describes some noteworthy details on how certain features are imple
 
-* `VersionedAddressBook#commit()` — Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` — Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` — Restores a previously undone address book state from its history.
+* `VersionedInventoryinator#commit()` — Saves the current inventory state in its history.
+* `VersionedInventoryinator#undo()` — Restores the previous inventory state from its history.
+* `VersionedInventoryinator#redo()` — Restores a previously undone inventory state from its history.
 
-These operations are exposed in the `Model` interface as `Model#commitAddressBook()`, `Model#undoAddressBook()` and `Model#redoAddressBook()` respectively.
+These operations are exposed in the `Model` interface as `Model#commitInventoryinator()`, `Model#undoInventoryinator()` and `Model#redoInventoryinator()` respectively.
 
 Given below is an example usage scenario and how the undo/redo mechanism behaves at each step.
 
-Step 1. The user launches the application for the first time. The `VersionedAddressBook` will be initialized with the initial address book state, and the `currentStatePointer` pointing to that single address book state.
+Step 1. The user launches the application for the first time. The `VersionedInventoryinator` will be initialized with the initial inventory state, and the `currentStatePointer` pointing to that single inventory state.
 
 ![UndoRedoState0](images/UndoRedoState0.png)
 
-Step 2. The user executes `delete 5` command to delete the 5th person in the address book. The `delete` command calls `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book state.
+Step 2. The user executes `del Bob’s 28th finger` command to delete the matching item in the inventory. The `del`
+command calls `Model#commitInventoryinator()`, causing
+ the modified state of the inventory after the `del Bob’s 28th finger` command executes to be
+ saved in the `inventoryinatorStateList`, and the `currentStatePointer` is shifted to the newly inserted inventory state.
 
 ![UndoRedoState1](images/UndoRedoState1.png)
 
-Step 3. The user executes `add n/David …​` to add a new person. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `addi Bob’s 6th regret -q 8` to add a new item. The `add` command also calls `Model#commitInventoryinator()`, causing another modified inventory state to be saved into the `inventoryinatorStateList`.
 
 ![UndoRedoState2](images/UndoRedoState2.png)
```

Comment:
same here

Reply:
Not known how implementation will be done, hence maintaining similar format as prior implementation for now.